src/combat.cpp

```diff
@@ -260,8 +259,7 @@ ReturnValue Combat::canDoCombat(Creature* caster, Tile* tile, bool aggressive)
 	if (aggressive && tile->hasFlag(TILESTATE_PROTECTIONZONE)) {
 		return RETURNVALUE_ACTIONNOTPERMITTEDINPROTECTIONZONE;
 	}
-
-	return g_events->eventCreatureOnAreaCombat(caster, tile, aggressive);
+	return tfs::events::creature::onAreaCombat(caster, tile, aggressive);
```

Comment:
I'm still unsure whether to use tfs::events or tfs::game::events. What do you think?

Reply:
`tfs::events`